src/AbstractActionHandler.ts

```diff
@@ -155,8 +155,9 @@ export abstract class AbstractActionHandler {
    *
    * @param candidateType   The incoming action's type
    * @param subscribedType  The type the Updater of Effect is subscribed to
+   * @param _payload        The payload of the incoming action.
```

Comment:
Any reason not to make this `payload`?

Reply:
Without `_`, there's a Typescript `noUnuserVariables` error. The `_` resolves this error, but then invokes a TSLint error, which I'm ignoring. I was being explicit about the TSLint error (`tslint:disable:variable-name`) but that put me over the 120 character limit, so I'd have to disable that rule too. Putting it on the above line breaks the JSDoc.